requirements.txt

```diff
@@ -7,6 +7,7 @@ django-bootstrap-form==3.4
 django-extensions==2.2.9
 django-mathfilters==1.0.0
 django-storages==1.9.1
+django-multiselectfield==0.1.12
```

Comment:
FYI, I'm opening an issue on deciding between the MultiSelectField and the ManyToMany with a Model (take wanted_assignments vs areas to see the difference)

Reply:
#199